src/tests/utils/ApiTestSteps.ts

```diff
@@ -319,14 +319,17 @@ export async function validateTxMAEventData(keyList: any, yotiMockID: any): Prom
 	}
 }
 
-export async function validateTxMAEvent(txmaEvent: any, keyList: any, yotiMockId: any): Promise<any> {
+export async function validateTxMAEvent(txmaEvent: any, keyList: any, yotiMockId: any, failedCheck: Boolean, vcData: any): Promise<any> {
 	let i:any;
 	for (i = 0; i < keyList.length; i++) {
 		const getObjectResponse = await HARNESS_API_INSTANCE.get("/object/" + keyList[i], {});
 	
 		if (getObjectResponse.data.event_name === txmaEvent) {
 			console.log(JSON.stringify(getObjectResponse.data, null, 2));
 			validateCriVcIssuedTxMAEvent(getObjectResponse.data, yotiMockId);
+			if (failedCheck = true){
```

Comment:
This assigns `failedCheck` to be true - do we instead want to check that it is true by using `if (failedCheck === true)` or `if (failedCheck)`?

Reply:
Updated to if (failedCheck)